packages/build-utils/src/types.ts

```diff
@@ -428,6 +430,7 @@ export interface BuildResultV2Typical {
   framework?: {
     version: string;
   };
+  crons?: Cron[];
```

Comment:
I think we want to remove this one? Can be added on the server-side if it's needed there.

```suggestion
```